### PR TITLE
Refactor kustomize config command structure

### PR DESCRIPTION
**Pull request by pwittrock**
- Create cfg, fn, live parent commands
- Publish commands as a kubectl plugin -- `kubectl-krm`
- Introduce `kustomize live` subcommand
- Delete `cmd/kubectl`

### CODE REVIEW

**Comment by pwittrock** [comment]
cc @chancez

File: cmd/config/go.mod [hunk 1]
```diff
@@ -23,4 +27,7 @@ require (
 //
 // releasing/releasemodule.sh will remove this line and set the require version
 // to the kyaml version specified in releasing/VERSIONS
-replace sigs.k8s.io/kustomize/kyaml v0.0.0 => ../../kyaml
+replace (
+	sigs.k8s.io/kustomize/kyaml v0.0.0 => ../../kyaml
+	sigs.k8s.io/kustomize/kyaml v0.1.13 => ../../kyaml
```
Comment (monopole) [inline cmd/config/go.mod:32]: please add a comment as to why both are needed
also, the Go-based releasing program is looking good; i'd like to delete the bash script and the VERSIONS file asap.

can you try the go program next time ?
try this:
```
(cd releasing/releasing; go run . list)
(cd releasing/releasing; go run . release kyaml minor)
```

haven't added a make target for it yet

Comment (pwittrock) [reply cmd/config/go.mod:35]: SGTM

Comment (pwittrock) [reply cmd/config/go.mod:35]: Thanks for the link to the new release tooling.  So glad to see it.

**Comment by pwittrock** [comment]
@monopole The release.go program works for kyaml, but needs to support updating the versions of the kustomize libraries.

e.g. the script should remove the `replace` statements, and set the `require` statements to be the latest version of the library (for kustomize libraries)

**Pull request merged**